### ISSUE TITLE
Problem with the sketchpad

### ISSUE THREAD

**Issue by Shahin-rmz** [gradio-app/gradio#391]
Hello I am doing a very basic Mnist recognition with Sketchpad of Gradio.
Train and test are as follows:
```python
(x_train, y_train), (x_test, y_test) = tf.keras.datasets.mnist.load_data()
x_train = x_train / 255.0, 
x_test = x_test / 255.
```
training is with keras sequential. with Relu and softmax activators.
```python
model = tf.keras.models.Sequential([
  tf.keras.layers.Flatten(input_shape=(28, 28)),
  tf.keras.layers.Dense(128,activation='relu'),
  tf.keras.layers.Dense(10, activation='softmax')
])
model.compile(optimizer='adam', loss='sparse_categorical_crossentropy', metrics=['accuracy'])
model.fit(x_train, y_train, validation_data=(x_test, y_test), epochs=6
```
here is my interface code:

```python
def classify(sketchpad):
    prediction = model.predict(sketchpad).tolist()[0]
    return {str(i): prediction[i] for i in range(10)}

label = gr.outputs.Label(num_top_classes=3)
interface = gr.Interface(fn = classify,inputs=sketchpad, outputs=label, live=True, capture_session=True)
interface.launch(debug=True)
```
The error is:

> Input 0 of layer "dense_6" is incompatible with the layer: expected axis -1of input shape to have value 784, but received input with shape (None, 28)
    
    Call arguments received:
      • inputs=tf.Tensor(shape=(None, 28), dtype=uint8)
      • training=False
      • mask=None
>

Maybe the pixels of sketchpad is not compatible with the number of pixels in dataset?

**Comment by abidlabs** [comment]
Hi @Shahin-rmz please replace the last cell block with this and it should work:

```python
def classify(input):
    prediction = model.predict(input.reshape(1, 28, 28)).tolist()[0]
    return {str(i): prediction[i] for i in range(10)}

label = gr.outputs.Label(num_top_classes=3)
interface = gr.Interface(fn = classify,inputs="sketchpad", outputs=label, 
live=True)
interface.launch(debug=True)
```

I've also updated the tutorial here: https://medium.com/swlh/a-gui-to-recognize-handwritten-digits-in-19-lines-of-python-fda715e525d0
